docs/0.intro.md

```diff
@@ -4,34 +4,36 @@ sidebar_position: 0
 title: ''
 ---
 
-## Who am I ?
+## Who am I?
+
 ---
 
 <h2 align="center">Hi 👋, I'm Shaurya Singhal 🇮🇳</h2>
-<h4 align="center">🤝 Welcome to Shaurya Notes/Blogs</h4>
+<h4 align="center">🤝 Welcome to Shaurya's Notes/Blogs</h4>
 <p align="center"> <img src="https://komarev.com/ghpvc/?username=jugshaurya&label=Profile Views&color=blue&style=plastic" alt="jugshaurya" /> </p>
 
 ### Experience/Achievements/Recent Failures
 
-- I am a **M.C.A 2rd Year Computer Science Student** @ University of Delhi, Department of Computer Science.
+- I am an **M.C.A 2<sup>nd</sup> Year Computer Science Student** @ the University of Delhi, Department of Computer Science.
```

Comment:
leave an as `a` and make t in the capital

Reply:
Will do